### ISSUE TITLE
jquery 2.x

### ISSUE THREAD

**Issue by ddurieux** [Alignak-monitoring-contrib/alignak-webui#4]
Perhaps it could be a good idea to update jquery to 2.x instead 1.x version ;)

**Comment by mohierf** [comment]
Yes, but currently it is not my 1st priority ... I am very easy with jQuery 1.x and I have many source code with this version ;)
